src/styles/variable.scss

```diff
@@ -0,0 +1,16 @@
+$day-size: 100px;
+$calendar-width: 712px;
+$month-length: 28, 29, 30, 31;
+$day-color: #eee;
+$padding: 10px;
+$border: 1px solid black;
+$gap: 10px;
+$days: (
+  'mon': 5.5,
```

Comment:
The value `5.5` is hardcoded for each day in the `$days` map. Ensure this value is appropriate or consider calculating it dynamically if needed.